client/src/pages/Profile/Profile.tsx

```diff
@@ -23,6 +25,24 @@ const SignOutButton = () => {
   )
 }
 
+const determineUserSkiSnowboardStatus = (
+  currentUserData?: UserAdditionalInfo
```

Comment:
also nit: prefer pass by value when possible (just including the skiier and snowboarding) so the function can't "accidentally" mutate the object

Reply:
Updated, let me know if that's better before I merge